scripts/server.js

```
@@ -1,38 +1,61 @@
+        headers: {
+          "Content-type": "application/json",
+          email: this.#email,
+          password: this.#password,
+        },
+      });
+      // console.log(await response.text());
+      const response_data = await response.json();
+      switch (response_data.status) {
+        case 200:
+          return "Enviado com sucesso";
+          break;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)